subxt/src/book/usage/mod.rs

```diff
@@ -10,12 +10,14 @@
 //! - [Constants](constants)
 //! - [Blocks](blocks)
 //! - [Runtime APIs](runtime_apis)
+//! - [Unstable Light Client](light_client)
 //!
 //! Alternately, [go back](super).
 
 pub mod blocks;
 pub mod constants;
 pub mod events;
+pub mod light_client;
```

Comment:
Probably we need to think about where is suitablke to put this in the book; I think it doesn't make sense here, but I can have a look at this properly though after this PR is merged so this is all good for now :)